db/seeds.rb

```diff
@@ -1,43 +1,16 @@
 # frozen_string_literal: true
 
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
-
-app_count = ENV['APP_COUNT'] || 100
-type_count = ENV['TYPE_COUNT'] || 10
-
-account = Account.new(:account_number => '1234')
-account.save!
+app_count = ENV['APP_COUNT'] || 10
+type_count = ENV['TYPE_COUNT'] || 5
 
 ActiveRecord::Base.transaction do
   app_count.times do |app_id|
-    app = Builder::App.build! do |app|
+    Builder::App.build! do |app|
       app.name "seed-app-#{app_id}"
-
-      Random.rand(type_count).times do |type_id|
+      event_type_count = Random.rand(type_count)
+      event_type_count.times do |type_id|
         app.event_type "seed-type-#{type_id}"
       end
     end
-
-    Builder::Filter.build!(account) do |filter|
-      filter.application(app.name)
-            .event_types(app.event_types.pluck(:name))
-      filter.severity.any!
-    end
   end
-
-  test_acc = Account.find_or_create_by(id: '00000000-0000-0000-0000-000000000000', account_number: '0000')
```

Comment:
I am using these objects for the end to end test. If you don't want them in seeds, can we move it to the test code itself with `find_or_create`?

Reply:
If you mean the task in `notifications.rake` this seems to already do a `find_or_create`. All tests seem to use factories, as far as I saw.

Reply:
I mean `find_or_create` for other objects - filter, endpoint e.t.c.

And yes, tests are using factories.

Reply:
`notifications.rake` uses `find_or_create` but only for the account. In order for the end to end test to work, we also need the endpoint and the filter.